### PR TITLE
Add test case for Getting from empty Hash

### PR DESCRIPTION
**Pull request by cnu**
In consistenthash, getting from an empty hash (before keys are added)
trying to Get a value should return an empty string.

Now has 100% coverage on consistenthash package

### CODE REVIEW

**Comment by cnu** [comment]
I signed it!

File: consistenthash/consistenthash_test.go [hunk 1]
```diff
@@ -34,6 +34,13 @@ func TestHashing(t *testing.T) {
 		return uint32(i)
 	})
 
+	// Get a value before any key is added to the hash
+	// Should yield an empty string
+	empty_value := hash.Get("empty key")
```
Comment (bradfitz) [inline consistenthash/consistenthash_test.go]: we don't use underscores in variable names in Go.

Also, what do you mean by "empty key"? The key isn't empty. Maybe just "foo"?

File: consistenthash/consistenthash_test.go [hunk 2]
```diff
@@ -34,6 +34,13 @@ func TestHashing(t *testing.T) {
...
+	if empty_value != "" {
+		t.Errorf("Expecting empty string, got %s", empty_value)
```
Comment (bradfitz) [inline consistenthash/consistenthash_test.go]: Go style for errors is:

    Func(input) = got; want foo

Also, use %q for strings when the string isn't what you expect. %q shows empty strings and strings with binary gibberish like newlines well.

I'd write this all as:

```go
    if got := hash.Get("foo"); got != "" {
          t.Errorf("Get on empty hash = %q; want empty string", got)
    }
```

**Comment by bradfitz** [comment]
Please squash these all together into one commit. As you make code review changes, squash again. We don't need to see corrections in the history. (Some projects like that, but not Go ones)

**Comment by cnu** [comment]
Have updated to use go style coding practices.

**Comment by cnu** [comment]
@bradfitz Sorry, didn't see your comment before commenting.
Would you prefer one commit for both the package's test cases? or should I separate them out and do a separate PR for the lru package?

**Comment by cnu** [comment]
I will go ahead and write more test cases for the lru package and then will squash them all into a single commit.

**Comment by cnu** [comment]
I have added more test cases and squashed the commits into one.
